### PR TITLE
[Docs] - add 0x prefix and change SuiAddress and ObjectID to lowercase

### PR DESCRIPTION
**Pull request by patrickkuo**
This is the doc changes for https://github.com/MystenLabs/sui/pull/1800.

I have added 0x prefix to all the addresses and object ID, also output of `objects` command now prints a table, I have included the change in docs as well.

### CODE REVIEW

File: doc/src/build/json-rpc.md [hunk 1]
```diff
@@ -153,7 +153,7 @@ The signing tool will create and print out the signature and public key informat
 You will see output resembling:
 ```shell
 2022-04-25T18:50:06.031722Z  INFO sui::sui_commands: Data to sign : VHJhbnNhY3Rpb25EYXRhOjoAAFHe8jecgzoGWyGlZ1sJ2KBFN8aZF7NIkDsM+3X8mrVCa7adg9HnVqUBAAAAAAAAACDOlrjlT0A18D0DqJLTU28ChUfRFtgHprmuOGCHYdv8YVHe8jecgzoGWyGlZ1sJ2KBFN8aZdZnY6h3kyWFtB38Wyg6zjN7KzAcBAAAAAAAAACDxI+LSHrFUxU0G8bPMXhF+46hpchJ22IHlpPv4FgNvGOgDAAAAAAAA
-2022-04-25T18:50:06.031765Z  INFO sui::sui_commands: Address : 51DEF2379C833A065B21A5675B09D8A04537C699
+2022-04-25T18:50:06.031765Z  INFO sui::sui_commands: Address : 0x51DEF2379C833A065B21A5675B09D8A04537C699
```
Comment (Clay-Mysten) [inline doc/src/build/json-rpc.md:156]: Should the letters be lower case here?

File: doc/src/build/wallet.md [hunk 2]
```diff
@@ -503,16 +506,15 @@ OPTIONS:
...
 This should give you output similar to the following:
 
 ```shell
-Owner: AddressOwner(k#66af3898e7558b79e115ab61184a958497d1905a)
-Version: 0
-ID: EEA4167BE074537F4A2879C7781D8EF4FFD651CC
-Readonly: false
+ID: 0x124bbde643189b573c98d05c092f4927225421d7
+Version: 1
+Owner: Account Address ( 0x62cd5bc220b28a34265bcb24995fb45a51d39832 )
```
Comment (Clay-Mysten) [inline doc/src/build/wallet.md:517]: Is this spacing in parenthesis correct?
( 0x62cd5bc220b28a34265bcb24995fb45a51d39832 )

**Review by Clay-Mysten — APPROVED** [review]
Thank you so much, Patrick! I have just a couple of comments/questions. Otherwise, LGTM! Adding Ade for another set of eyes. Thanks again!

File: doc/src/build/json-rpc.md [hunk 3]
```diff
@@ -201,11 +201,11 @@ curl --location --request POST $SUI_RPC_HOST \
               "method": "sui_moveCall",
               "params": [
                   "{{owner_address}}",
-                  "0000000000000000000000000000000000000002",
+                  "0x2",
```
Comment (oxade) [inline doc/src/build/json-rpc.md:204]: Do we allow short form everywhere?

Comment (patrickkuo) [reply doc/src/build/json-rpc.md:204]: yes, I have done some test manually and a unit test here https://github.com/MystenLabs/sui/blob/d4d894e7dbb1e9a0d763649a3bf1d85fc55d4710/sui_types/src/unit_tests/base_types_tests.rs#L198-L215

Should be allowed everywhere

**Pull request merged**